integration/spark2/src/test/scala/org/apache/spark/carbondata/restructure/vectorreader/AddColumnTestCases.scala

```diff
@@ -42,26 +43,55 @@ class AddColumnTestCases extends Spark2QueryTest with BeforeAndAfterAll {
       "('DICTIONARY_EXCLUDE'='charField', 'DEFAULT.VALUE.charfield'='def')")
     sql(s"LOAD DATA LOCAL INPATH '$resourcesPath/restructure/data1.csv' INTO TABLE addcolumntest " +
         s"options('FILEHEADER'='intField,stringField,charField,timestampField,decimalField')")
+    sql("CREATE TABLE hivetable stored as parquet select * from addcolumntest")
   }
 
   test("test like query on new column") {
+    sqlContext.setConf("carbon.enable.vector.reader", "true")
```

Comment:
true should means vector reader, false means row reader, actually it is the main difference between the 2 test class packages which need to merge